styles.css

```diff
@@ -2,33 +2,78 @@
     padding: 0;
     margin: 0;
 }
+
 .body-wrapper {
     background-color: #587aec;
     color: #ffffff;
+    display: flex;
+    flex-direction: column;
     align-items: center;
+    min-height: 100vh;
```

Comment:
this is not needed. It will cause issues as the page grows. The page itself behaves in a really good way without explicitly defining this.